Gemfile

```diff
@@ -27,6 +27,7 @@ gem 'will_paginate'
 gem 'acts-as-taggable-on', '~> 6.0'
 gem 'omniauth-census', git: "https://github.com/turingschool-projects/omniauth-census"
 gem 'omniauth-github', github: 'omniauth/omniauth-github', branch: 'master'
+gem 'sendgrid-ruby'
```

Comment:
we are not actually using this yet, but it was in the lesson and I think it comes into play in the next email related user story?